### ISSUE TITLE
Order devices by bluetooth strength

### ISSUE THREAD

**Issue by bambibrewer** [BirdBrainTechnologies/flutter-kindle-sttr2#70]
When scanning, it would be a good idea to order the Flutter by bluetooth strength. That way, the ones close to you would show up near the top of the list, which would be helpful in a classroom.

**Comment by tasota** [comment]
Also @achen783 I forgot to mention today but we'll definitely want this for both apps, so be sure to work from the `org.cmucreatelab.kindle.fluttersttr2.activities.scan` package (in the main directory instead of the subclasses in the arithmetic or place_value directory)

**Comment by tasota** [comment]
@bambibrewer does something like this already exist in BirdBlox or some other app? Emily and I were wondering.

**Comment by bambibrewer** [comment]
Absolutely! This is implemented in the BirdBlox-Android-Support repository.

**Comment by bambibrewer** [comment]
Let me know if you need me to help you find where.

**Comment by tasota** [comment]
Thank you bambi. I think I found it, though I'm not sure if this will help with sorting a list in android since it looks like this is all converted into json and sent to a webview potentially.

@achen783 I think it's `ScanResult.getRssi()` that you'll be interested in, but again I'm not sure if there is any sorting logic in this code unfortunately. https://github.com/BirdBrainTechnologies/BirdBlox-Android-Support/blob/master/app/src/main/java/com/birdbraintechnologies/birdblox/Bluetooth/BluetoothHelper.java

**Comment by bambibrewer** [comment]
You are correct, Mike - the sorting is done in JSON. The Android code is only useful for figuring out how to get the signal strength, which clearly you have done!
